lib/Transforms/Scalar/UnifyAssumes.cc

```diff
@@ -57,6 +59,18 @@ bool isAssumeCall(const CallInst &ci, seahorn::SeaBuiltinsInfo &SBI) {
     return true;
   }
 }
+
+bool isAssertCall(const CallInst &ci, seahorn::SeaBuiltinsInfo &SBI) {
```

Comment:
as discussed, move  `SBI` to `m_SBI` field and set it in `runOnFunction`

Reply:
I don't know if you saw my previous comment - https://github.com/priyasiddharth/seahorn/pull/2#discussion_r475896092

Reply:
https://github.com/priyasiddharth/seahorn/pull/2#discussion_r475901061

Reply:
see my reply to your reply

Reply:
Fixed